client/my-sites/upgrades/domain-management/edit/test/mapped-domain.js

```diff
@@ -26,7 +27,8 @@ describe( 'mapped-domain', () => {
 				name: 'neverexpires.com',
 				expirationMoment: null
 			},
-			settingPrimaryDomain: false
+			settingPrimaryDomain: false,
+			translate: identity
```

Comment:
This would be better if someone would use the other `export`, but now that I think about it, would be better if we didn't set it in there, so no one would use it `unlocalized`. :D